src/main/java/com/dodatabase/movie_backend/WebClientConfig.java

```diff
@@ -22,12 +22,9 @@ public WebClientConfig(ApiKey apiKey) {
 
     @Bean
     public WebClient movieApiClient() {
+        String HOST = KMDB_HOST + "&ServiceKey=" + apiKey.getKey();
```

Comment:
Just paste strings.
Shoud change it to use some modules which can connect cleanly.